charts/prometheus/README.md

```diff
@@ -65,6 +65,12 @@ helm upgrade [RELEASE_NAME] prometheus-community/prometheus --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 25.0
+
+The `server.remoteRead[].url` and `server.remoteWrite[].url` fields now support templating. Allowing for `url` values such as `https://{{ .Release.Name }}.example.com`.
+
+Any entries in these which previously included `{{` or `}}` must be escaped with `{{ "{{" }}` and `{{ "}}" }}` respectively. Entries which did not previously include the template-like syntax will not be affected.
+
```

Comment:
```suggestion
```

As an upgrade note is not to be made for a minor change, please, remove it from README (upgrade notes include only actions users have to take in order to successfully upgrade their release). I'd suggest instead to insert a short comment in the values file that both the fields support templating `url`.